src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -732,9 +732,9 @@ class JsonPrinter : public Printer {
                 d_ostream << ',';
             }
 
-            d_ostream << "\n    {\"leaseId\": \"" << it->leaseId()
-                      << "\", \"sequenceNumber\": \"" << it->sequenceNumber()
-                      << "\"}";
+            d_ostream << "\n    {\"leaseId\": " << it->leaseId()
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.